bin/eds-import-from-figma.js

```diff
@@ -0,0 +1,258 @@
+#!/usr/bin/env node
+(async function () {
+  const fs = require('fs');
+  const jsonfile = require('jsonfile');
+  const yargs = require('yargs/yargs');
+
+  // eslint-disable-next-line import/extensions
+  const { hideBin } = require('yargs/helpers');
+  const { prompt } = require('enquirer');
+  const at = require('lodash/at');
+  const set = require('lodash/set');
+
+  const { default: ora } = await import('ora');
+  const chalk = require('chalk');
```

Comment:
These are the lib.s used in many popular tools like `husky` to print animated spinners, change the color of text, etc.